test/Microsoft.Dnx.Runtime.Tests/NuGet/SemanticVersionTest.cs

```diff
@@ -31,7 +31,7 @@ public static IEnumerable<object[]> NormalizedVersionStringTestData
         [MemberData(nameof(NormalizedVersionStringTestData))]
         public void TestGetNormalizeVersionString(SemanticVersion version, string expectation)
         {
-            Assert.Equal(expectation, version.GetNormalizedVersionString());
+            Assert.Equal(expectation, version.ToString());
```

Comment:
Could you also add a test case for `1.0-beta` -> `1.0.0-beta` - doesn't seem to be covered?